pkg/driver/driver_aws.go

```diff
@@ -305,3 +305,17 @@ func (d *AWSDriver) decodeMachineID(id string) string {
 	splitProviderID := strings.Split(id, "/")
 	return splitProviderID[len(splitProviderID)-1]
 }
+
+// GetVolNames parses volume names from pv specs
+func (d *AWSDriver) GetVolNames(specs []corev1.PersistentVolumeSpec) ([]string, error) {
+	names := []string{}
+	for _, spec := range specs {
```

Comment:
Same shallow copy issue as [above](https://github.com/gardener/machine-controller-manager/pull/275/files#diff-face99aebd3cd79577003f7df747c863R335) but with less impact because the number of volumes per pod will be less.